numba_cuda/numba/cuda/tests/cudadrv/test_nvvm_driver.py

```diff
@@ -138,9 +138,9 @@ def test_nvvm_warning(self):
 class TestArchOption(unittest.TestCase):
     def test_get_arch_option(self):
         # Test returning the nearest lowest arch.
-        self.assertEqual(nvvm.get_arch_option(5, 3), "compute_53")
         self.assertEqual(nvvm.get_arch_option(7, 5), "compute_75")
         self.assertEqual(nvvm.get_arch_option(7, 7), "compute_75")
```

Comment:
+1 should we make these more programmatic to guard against future failures?

Reply:
That could be done, but as it's changed once in 5 years I'm not sure it's worth the effort to programmatically figure out valid cases for these tests. (I'm also not sure this will continue to exist once we're using `cuda.core` / `cuda.bindings` more heavily.)